src/odemis/gui/util/widgets.py

```diff
@@ -126,8 +126,9 @@ def _connect(self, init):
 
     def disconnect(self):
         logging.debug("Disconnecting VigilantAttributeConnector")
-        for event in self.change_events:
-            self.value_ctrl.Unbind(event, source=self.value_ctrl, handler=self._on_ctrl_value_change)
+        if self.value_ctrl:
```

Comment:
Isn't this always None since disconnect() is now called in the end of _on_ctrl_destroy?